java/README.md

```diff
@@ -0,0 +1,17 @@
+### plaid-java quickstart
+
+[Quickstart guide](https://plaid.com/docs/quickstart)
+
+``` bash
+git clone https://github.com/plaid/quickstart.git
+cd quickstart/java
+mvn clean install
+
+# Fill in your Plaid API keys (client ID, secret, public_key)
+# to test!
+PLAID_CLIENT_ID=[CLIENT_ID] \
+PLAID_SECRET=[SECRET] \
+PLAID_PUBLIC_KEY=[PUBLIC_KEY] \
+java -jar target/quickstart-1.0-SNAPSHOT.jar server config.yml
```

Comment:
can we document which version of java we need for this?

Reply:
added version above

Reply:
can we make our quick start target at least 8 or higher (since 8 is the current LTS version)?

Reply:
updated `README.md` and `pom.xml` with java v1.8